src/bb/image-io/rt_u3v.h

```diff
@@ -484,6 +484,22 @@ class U3V {
         return err_;
     }
 
+    void ValidateUserInput(int32_t num_detected_device, char* dev_id){
+        if (num_detected_device < num_sensor_){
+            log::info("{} device is found; but the num_sensor is set to {}", num_detected_device, num_sensor_);
+            throw std::runtime_error("Device number is not match, please set num_device again");
+        }
+        frame_sync_ = num_sensor_ > 1 ? frame_sync_ : false;
+
+        if (num_detected_device > num_sensor_ && dev_id == nullptr) {
+            log::info("{} devices are found; The first {} device is selected", num_detected_device, num_sensor_);
+        }
+
+        log::info("Creating U3V instance with {} sensors...", num_sensor_);
+        log::info("Acquisition option::{} is {}", "frame_sync_", frame_sync_);
+        log::info("Acquisition option::{} is {}", "realtime_display_mode_", realtime_display_mode_);
+    }
+
     GError* CommandAcquisitionModeContdStart(){
```

Comment:
Do we have a specific policy for the error handling? It looks function is expected to return GError *, but it actually throws an exception in many cases.

Reply:
Updated in [c654c02](https://github.com/fixstars/ion-kit/pull/324/commits/c654c02858f79ed72af1cf0b998e98b59f101495)